helpers.js

```diff
@@ -56,8 +56,18 @@ function getDistance(origin, destination) {
 // returns closure that sorts list by key
 function sorterBy(key, desc){
     return function(a,b) {
+        // Put the relevant variables into i and j
+        i = a[key]
+        j = b[key]
+
+        // Convert i and j to numbers if they can be converted to them
+        if( !isNaN(i) && !isNaN(j) ) {
+            i = Number(i)
+            j = Number(j)
+        }
+
         r = 1
-        if ( b[key] > a[key] ) {
+        if (i > j) {
```

Comment:
Can you please change to `i < j`, the sorting orientation got flipped (clashes with the arrow)
 
![image](https://user-images.githubusercontent.com/872894/91048209-95cd8100-e61b-11ea-8ace-fe8cea9557ee.png)

Reply:
Fixed it in 5638b33d80d23e197060a464524300b6d98d1025